src/bootstrap.ts

```diff
@@ -37,24 +37,19 @@ const application = (): Promise<Server> =>
     },
     swaggerOptions: {
```

Comment:
It will be better if you have an excellent example of the options that this new swagger package expects.

Reply:
Again I am not sure on the core expectations here. Does this help [hapi-swagger](https://github.com/glennjones/hapi-swagger/blob/master/usageguide.md)

Reply:
@babaalaja `bootstrap.ts` is a sample app that uses this **@ctt/crud-api** package. What I mean is that you should add the expected options that the `hapi-swagger` package accepts. 

I'm not sure what we have there now still works because it is an option for a different swagger package.

Reply:
@Ehbraheem It works, the package does not need the accompanying  Swagger UI package that the old Swagger Package required. Did you check **/documentation** when your ran `yarn bootstrap`

Reply:
No, I didn't. Please record a video that shows this 🙏

Reply:
https://drive.google.com/file/d/1eDDBfZnqvzJGyLcRMh8UJSj7IMhK_Ume/view?usp=sharing